packages/rs-drive-abci/src/platform_types/validator_set/v0/mod.rs

```diff
@@ -306,61 +308,3 @@ impl ValidatorSet {
         })
     }
 }
```

Comment:
Logically everything related to ValidatorSetUpdate must be in ABCI. Theoretically, we should be able to plug execution to any interface like ABCI so it could be many implemented there. This module shouldn't know about these implementations.